core/math/vector2.h

```diff
@@ -241,14 +241,14 @@ _FORCE_INLINE_ bool Vector2::operator!=(const Vector2 &p_vec2) const {
 	return x != p_vec2.x || y != p_vec2.y;
 }
 
-Vector2 Vector2::lerp(const Vector2 &p_to, const real_t p_weight) const {
+_FORCE_INLINE_ Vector2 Vector2::lerp(const Vector2 &p_to, const real_t p_weight) const {
```

Comment:
Are you sure you need the `_FORCE_INLINE_` in both the declaration and the definition? I haven't really tested this, but I see a lot of existing code just uses it in the declaration.

Reply:
https://github.com/godotengine/godot/pull/87093#discussion_r1450150604

Seems like the best course of action would be to use inlining in the header file coupled with a definition and remove that method from the `.cpp` file.

Reply:
I removed `_FORCE_INLINE_` from the declaration and added it to the definition